backend/src/main/java/de/bund/digitalservice/ris/norms/application/service/TimeMachineFunctions.java

```diff
@@ -0,0 +1,118 @@
+package de.bund.digitalservice.ris.norms.application.service;
+
+import java.util.Optional;
+import org.w3c.dom.Document;
+import org.w3c.dom.Node;
+
+/**
+ * Namespace for business Logics related to "time machine" functionality, i.e. to applying LDML.de
+ * "modifications" to LDML.de files. For details on LDML.de modifications, cf.
+ * https://gitlab.opencode.de/bmi/e-gesetzgebung/ldml_de/-/tree/main/Spezifikation?ref_type=heads
+ */
+public class TimeMachineFunctions {
+
+  // prevent default constructor
+  private TimeMachineFunctions() {}
```

Comment:
I suppose you want to prevent a default constructor because all functions underneath are static? Why not just using a @Service or @Component annotation from spring?